src/saga_action_generic.rs

```diff
@@ -105,7 +105,18 @@ pub trait Action<UserType: SagaType>: Debug + Send + Sync {
     /**
      * Executes the action for this saga node, whatever that is.  Actions
      * function like requests in distributed sagas: critically, they must be
-     * idempotent.  They should be very careful in using interfaces outside of
+     * idempotent. This means that multiple calls to the action have the
+     * same result on the system as a single call, although the action is not
+     * necessarily required to return the same result.
+     *
+     * As an example, generating a UUID to represent an object is a common saga
+     * action: if called repeatedly, it may generate different results, but it
+     * has the no side effects on the rest of the system. Similarly, using a
```

Comment:
```suggestion
     * has no side effects on the rest of the system. Similarly, using a
```

(may need to rewrap after that)

Reply:
Done